docs/api/request.md

```diff
@@ -117,10 +117,10 @@ this.request.charset
 
   For example "color=blue&size=small":
 
-```js
+```json
 {
-  color: 'blue',
-  size: 'small'
+  "color": "blue",
+  "size": "small"
```

Comment:
I prefer this without the extra quotes, will the markdown linter not let us do that?